src/test/integration/boto3-sqs/tests/test_boto3_sqs.py

```diff
@@ -0,0 +1,71 @@
+import unittest
```

Comment:
can you also verify direct access to "Messages"
e.g (["Messages"][0] - break trace?

Reply:
Done in 904b96a